### PR TITLE
[FB-001] Fibbonaci Numbers 

### PR DESCRIPTION
**Pull request by Wlxty**

### CODE REVIEW

File: Data/Fibbonaci.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package Data
+
+func Fibbonaci(size int) []int{
+	if size < 1{
+		return nil
+	}else{
+		array := []int{1}
+		for num := 0; num < size; num++ {
+			if num == 0 {
```
Comment (sgrzywna) [inline Data/Fibbonaci.go]: I think it will be better to move `if` outside the loop.

**Comment by sgrzywna** [comment]
I can see that you are not using `gofmt`. You should.

And if I remember correctly 0 is a valid number in Fibonacci sequence.

**Comment by Wlxty** [comment]
@sgrzywna I've deleted if statement from for loop.
I'm going to build some tests in a moment

File: Data/Fibbonaci.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package Data
+
+func Fibbonaci(size int) []int {
+
+	if size <= 0 {
+		return nil
```
Comment (sgrzywna) [inline Data/Fibbonaci.go]: There is no need to use `else` after `return` statement.

https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting

File: main.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"Fibbonaci/Data"
```
Comment (sgrzywna) [inline main.go]: Package names should be lower case.

https://github.com/uber-go/guide/blob/master/style.md#package-names

Comment (sgrzywna) [inline main.go]: I'm not sure why you use `Data` package? Just `Fibbonaci` should be enough.

BTW should be `Fibonacci` (two `c`).

File: Data/go.mod [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module Data
```
Comment (sgrzywna) [inline Data/go.mod]: `mod` and `sum` files for a such small project should be stored in top directory (together with `main.go`).

**Comment by sgrzywna** [comment]
Storing compiled version of the application (binaries in general) in the repository is rather unusual.

Usually binary _artifacts_ are saved into the dedicated storage. For example, for github you can use [releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).

**Pull request merged**